client/my-sites/marketing/tools/google-adwords.tsx

```diff
@@ -0,0 +1,113 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import page from 'page';
+import React, { Fragment, FunctionComponent } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { addQueryArgs } from 'lib/url';
+import Button from 'components/button';
+import { getCurrentUserCountryCode } from 'state/current-user/selectors';
+import { getSelectedSite } from 'state/ui/selectors';
+import GoogleVoucherDetails from 'my-sites/checkout/checkout-thank-you/google-voucher';
+import { isPremium, isBusiness, isEcommerce, isEnterprise } from 'lib/products-values';
+import isSiteAtomic from 'state/selectors/is-site-automated-transfer';
+import MarketingToolsFeature from './feature';
+import { PLAN_PREMIUM } from 'lib/plans/constants';
+import QuerySiteVouchers from 'components/data/query-site-vouchers';
+import { recordTracksEvent as recordTracksEventAction } from 'state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface ConnectedProps {
+	isAtomic: boolean;
+	isJetpack: boolean;
+	isPremiumOrHigher: boolean;
+	recordTracksEvent: () => void;
+	showCard: boolean;
+	siteId: number | null;
+	siteSlug: string | null;
+}
+
+export const MarketingToolsGoogleAdwordsFeature: FunctionComponent< ConnectedProps > = ( {
+	isAtomic,
+	isJetpack,
+	isPremiumOrHigher,
+	recordTracksEvent,
+	showCard,
+	siteId,
+	siteSlug,
+} ) => {
+	const translate = useTranslate();
+	if ( ! showCard || ( isJetpack && ! isAtomic ) || ! siteId || ! siteSlug ) {
+		return null;
+	}
+
+	const handleUpgradeClick = () => {
+		recordTracksEvent( 'calypso_marketing_tools_adwords_plan_upgrade_button_click' );
+		page( addQueryArgs( { plan: PLAN_PREMIUM }, `/plans/${ siteSlug }` ) );
+	};
+
+	const renderButton = () => {
+		if ( isPremiumOrHigher ) {
+			return <GoogleVoucherDetails />;
+		}
+		return <Button onClick={ handleUpgradeClick }>{ translate( 'Upgrade To Premium' ) }</Button>;
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'Upgrade to Premium' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).